phik/simcore/__init__.py

```diff
@@ -1,6 +1,10 @@
 import importlib.util
 
-_ext_spec = importlib.util.find_spec("phik.lib._phik_simulation_core")
+try:
```

Comment:
Why is this needed? The `_ext_spec` should be `None` if it can't be found

Reply:
If `phik.lib` can't be found then this would return a `ModuleNotFoundError`.
importlib.util.find_spec will only return None if there is a module named 'phik.lib' but it doesn't contain `_phik_simulation_core`.
So given that this is trying to detect if the compiled CPP code exists this probably makes sense? If _phik_simulation_core hasn't been compiled then the `lib` folder probably won't exist and if the lib folder isn't there then the importlib.util.find_spec will end up as a ModuleNotFoundError rather than None

Though I'm not sure what this has to do with the change from mvn to qmvn

Reply:
Thanks for answering, clear. 
(It has nothing to do with the migration. Different commit, but in same PR review.)